pineappl_py/src/subgrid.rs

```diff
@@ -142,6 +143,53 @@ impl PySubgridParams {
     }
 }
 
+/// PyO3 wrapper to :rustdoc:`pineappl::subgrid::Mu2 <subgrid/struct.Mu2.html>`
+#[pyclass]
+#[repr(transparent)]
+pub struct PyMu2 {
+    pub(crate) mu2: Mu2,
```

Comment:
Is `pub(crate)` really needed?

Reply:
No, since you have `{set_}{ren,fac}` accessors down below, you do not need any other way to access.

Reply:
[ce61075](https://github.com/NNPDF/pineappl/pull/222/commits/ce610751d38c27a588a0c8592579186d89497bc0)